frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/common.ts

```diff
@@ -5,5 +5,5 @@ export const commonFlows = {
     cy.byLegacyTestID('item-filter').type('ocs');
     cy.byTestOperatorRow('OpenShift Container Storage').click();
   },
-  checkAll: () => cy.get('input[name=check-all]'),
+  checkAll: cy.get('input[name=check-all]'),
```

Comment:
```suggestion
  checkAll: () => cy.get('input[name=check-all]'),
```
We cannot reference cy directly as an object property because it's not present in this scope. Hence we need to use it as a function.